src/actions/action.ts

```diff
@@ -36,7 +36,7 @@ export interface ActionSpec {
    * Can be defined to display an icon
```

Comment:
The tooltip on hover of the menu item seems very ugly in this small menu. And 100% necessary, it just repeats the text